pxt.json

```diff
@@ -28,7 +29,7 @@
     "supportedTargets": [
         "microbit"
     ],
-    "preferredEditor": "blocksprj",
+    "preferredEditor": "tsprj",
```

Comment:
The changes in c5cf38d0e4507dfb378a276904c2a4b5e5379a27 have been made so that we can edit the test code directly in TS and blocks when importing the GH repo into MakeCode, which is helpful for testing.

The main.blocks file I've added is an empty blocks project, so I've changed the default view to start in the TS code editor, that way when the user presses the "block" view it then converts the main.ts code into blocks. 
If this entry was left as `blocksprj` then it would load the empty blocks project and the TS code would be lost when switching modes.

We can keep the main.blocks file empty, as syncing the blocks XML to the TS changes is a tedious process.